rero_ils/modules/ill_requests/jsonschemas/ill_requests/ill_request-v0.0.1.json

```
@@ -0,0 +1,223 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "ILLRequest",
+  "description": "JSON schema for ill_request.",
+  "additionalProperties": false,
+  "propertiesOrder": [
+    "document",
+    "pickup_location",
+    "status",
+    "copy",
+    "pages",
+    "found_in",
+    "note"
+  ],
+  "required": [
+    "$schema",
+    "pid",
+    "status",
+    "patron",
+    "pickup_location",
+    "copy",
+    "document"
+  ],
+  "properties": {
+    "$schema": {
+      "title": "Schema",
+      "description": "Schema to validate ILL request records against.",
+      "type": "string",
+      "minLength": 9,
+      "default": "https://ils.rero.ch/schema/ill-requests/ill-request-v0.0.1.json"
+    },
+    "pid": {
+      "title": "ILL request ID",
+      "type": "string",
+      "minLength": 1
+    },
+    "status": {
+      "title": "Status",
+      "type": "string",
+      "enum": [
+        "pending",
+        "validated",
+        "denied",
+        "closed"
+      ],
+      "default": "pending",
+      "form": {
+        "type": "selectWithSort",
+        "wrappers": [
+          "form-field-horizontal"
+        ],
+        "options": [
+          {
+            "value": "pending",
+            "label": "pending"
+          },
+          {
+            "value": "validated",
+            "label": "validated"
+          },
+          {
+            "value": "denied",
+            "label": "denied"
+          },
+          {
+            "value": "closed",
+            "label": "closed"
+          }
+        ],
+        "templateOptions": {
+          "selectWithSortOptions": {
+            "order": "label"
+          }
+        }
+      }
+    },
+    "copy": {
+      "title": "Copy",
+      "description": "Is this request for a copy request or not?",
```

I think that my proposition wasn't so good, maybe it's better like this:
```suggestion
      "description": "Define if the request is for a copy or an actual document.",
```

---

done